rd_ui/app/views/query.html

```diff
@@ -90,7 +90,7 @@
             <div class="schema-browser">
                 <div ng-repeat="table in schema | filter:schemaFilter track by table.name">
                     <div class="table-name" ng-click="table.collapsed = !table.collapsed">
-                        <i class="fa fa-table"></i> <strong><span title="{{table.name}}">{{table.name}}</span></strong>
+                        <i class="fa fa-table"></i> <strong><span title="{{table.name}}">{{table.name}} ({{table.size}})</span></strong>
```

Comment:
The size should only be available if there is actually size... Something like this: `<span ng-if="table.size"> ({{table.size}})</span>`.